src/components/ClipControls/ClipControls.jsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components";
 import { Range } from "rc-slider";
 import "rc-slider/assets/index.css";
 
-import Section from "./Section.jsx";
+import Section from "../Section/Section.jsx";
```

Comment:
```suggestion
import Section from "../Section";
```
Can you do this if you add an index to the Section folder and make Section the default epxort?